integration_tests/two_sided_pay_as_clear_market.feature

```diff
@@ -24,11 +24,11 @@ Feature: Two sided pay_as_clear market tests
   Scenario: Test offers, bids and market-clearing-rate are exported
      Given we have a scenario named two_sided_pay_as_clear/default_2a
      And d3a is installed
-     When we run the d3a simulation on console with two_sided_pay_as_clear.default_2a
+     When we run the d3a simulation on console with two_sided_pay_as_clear.default_2a for 24 hrs
```

Comment:
Minor, but how about decreasing the simulation runtime here as well to save time in the tests?